src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/service/UserGroupService.kt

```diff
@@ -113,7 +113,7 @@ class UserGroupService(
 
       telemetryClient.trackEvent(
         "ExternalUserGroupAddSuccess",
-        mapOf("userId" to userId.toString(), "group" to groupFormatted, "admin" to authenticationFacade.getUsername()),
+        mapOf("userId" to userId.toString(), "username" to userRepository.findById(userId)!!.getUserName(), "group" to groupFormatted, "admin" to authenticationFacade.getUsername()),
         null,
```

Comment:
At this  point we have already done 'userRepository.findById(userId)?.let { user ->'
So we should just be able do  'user.getUserName()' rather than having to back to the db

Reply:
Updated